Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1916,19 +1912,12 @@ CGImageRef CGBitmapContextGetImage(CGContextRef context) {
     return ((__CGBitmapContext*)context)->Impl().image.get();
 }
 
-CGContextRef _CGBitmapContextCreateWithTexture(
-    int width, int height, float scale, DisplayTexture* texture, DisplayTextureLocking* locking) {
-    CGImageRef newImage = nullptr;
-    __CGSurfaceInfo surfaceInfo = _CGSurfaceInfoInit(width, height, _ColorARGB);
-    newImage = new CGGraphicBufferImage(surfaceInfo, texture, locking);
-
-    ComPtr<ID2D1RenderTarget> renderTarget = newImage->Backing()->GetRenderTarget();
-    renderTarget->SetDpi(96 * scale, 96 * scale);
-
+CGContextRef  _CGBitmapContextCreateWithRenderTarget(ID2D1RenderTarget* renderTarget, CGImageRef img) {
+    RETURN_NULL_IF(!renderTarget);
+    //TODO: do it for image?
     __CGBitmapContext* context = __CGBitmapContext::CreateInstance(kCFAllocatorDefault);
-    __CGContextInitWithRenderTarget(context, renderTarget.Get());
-
-    context->Impl().image.reset(newImage); // Consumes +1 reference.
+    __CGContextInitWithRenderTarget(context, renderTarget);
+    context->SetImage(img);// Consumes +1 reference.
```

Comment:
and then remove the comment here.

---
In reply to: [87101477](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r87101477) [](ancestors = 87101477)